### PR TITLE
Commit #1. magnetic_induction_of_wire

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests.

### CODE REVIEW

File: symplyphysics/laws/electricity/magnetic_induction_of_wire.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless
+)
+from sympy.physics.units import magnetic_constant
+
+# Description
+## Let there be an infinite thin conductor. Then the magnetic field created by the current in
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/magnetic_induction_of_wire.py:14]: Add geometry of conductor to conditions.

File: test/electricity/magnetic_induction_of_wire_test.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (units, SI, convert_to, Quantity, errors)
+from symplyphysics.laws.electricity import magnetic_induction_of_wire as induction_law
+
+# Description
+## For a distance of 0.5 meters from the conductor and a current of 20 A, the magnetic field induction is 8 tesla.
```
Comment (blackyblack) [inline test/electricity/magnetic_induction_of_wire_test.py:7]: 8 tesla from just 20 amperes?
Check your numbers.

File: symplyphysics/laws/electricity/magnetic_induction_of_wire.py (hunk 1 above)
Comment (Zulusssss) [reply symplyphysics/laws/electricity/magnetic_induction_of_wire.py:14]: > Add geometry of conductor to conditions.

Does it mean just to describe the geometry of the conductor in the description?

Comment (blackyblack) [reply symplyphysics/laws/electricity/magnetic_induction_of_wire.py:14]: It seems the law depends on it. So conditions should contain geometry of conductor.

Comment (Zulusssss) [reply symplyphysics/laws/electricity/magnetic_induction_of_wire.py:14]: > It seems the law depends on it. So conditions should contain geometry of conductor.

Geometry requirements - the wire is infinitely thin. This is the only requirement for geometry. I described this requirement in the description.

Comment (blackyblack) [reply symplyphysics/laws/electricity/magnetic_induction_of_wire.py:14]: Yes, but I ask to put it into conditions.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**